internal/impl/openai/chat_processor.go

```diff
@@ -587,6 +602,21 @@ func (p *chatProcessor) Process(ctx context.Context, msg *service.Message) (serv
 			Content: s,
 		})
 	}
+	if p.history != nil {
+		msg, err := msg.BloblangQuery(p.history)
```

Comment:
only thing would be if history should be a storage query to postgres or whatever memory the user configured.

Reply:
Right now that will have to be done manually, we can add that on in another PR

Reply:
Provided an example of manually hooking up storage